src/networks/mod.rs

```diff
@@ -249,6 +252,13 @@ impl ChainConfig {
             breeze_gas_tamping_duration: BREEZE_GAS_TAMPING_DURATION,
             // 1 year on mainnet
             fip0081_ramp_duration_epochs: 365 * EPOCHS_IN_DAY as u64,
+            f3_bootstrap_epoch: -1,
+            f3_initial_power_table: Default::default(),
+            f3_mainfest_server: Some(
```

Comment:
These values are from Lotus but I suspect `f3_mainfest_server` will be deprecated once F3 is fully bootstrapped to avoid single point network dependencies

Reply:
Done.